src/resources/raw.ts

```diff
@@ -0,0 +1,228 @@
+
+  /**
+   * [Insert rows into a table](https://doc.cognitedata.com/api/v1/#operation/postRows)
+   *
+   * ```js
+   * await client.raw.insertRows('My company', 'Customers' [{ key: 'customer1', columns: { 'First name': 'Steve', 'Last name': 'Jobs' } }]);
+   */
+  insertRows: (
+    databaseName: string,
+    tableName: string,
+    items: RawDBRow[],
+    // tslint:disable-next-line:bool-param-default
+    ensureParent?: boolean
+  ) => Promise<{}>;
+
+  /**
+   * [Retrieve a single row from a table](https://doc.cognitedata.com/api/v1/#operation/getRow)
+   *
+   * ```js
+   * await client.raw.retrieveRow('My company', 'Customers', 'customer1');
+   */
+  retrieveRow: (
+    databaseName: string,
+    tableName: string,
+    rowKey: string
+  ) => Promise<RawDBRow>;
+
+  /**
+   * [Delete rows in a table](https://doc.cognitedata.com/api/v1/#operation/deleteRows)
+   *
+   * ```js
+   * await client.raw.deleteRows('My company', 'Customers', [{key: 'customer1'}]);
+   */
+  deleteRows: (
+    databaseName: string,
+    tableName: string,
+    items: RawDBRowKey[]
+  ) => Promise<{}>;
+}
+
+/** @hidden */
+export function generateRawObject(
+  project: string,
+  instance: AxiosInstance,
+  map: MetadataMap
+): RawAPI {
+  const path = projectUrl(project) + '/raw/dbs';
+  const tablePath = (databaseName: string) =>
+    `${path}/${encodeURIComponent(databaseName)}/tables`;
+  const rowPath = (databaseName: string, tableName: string) =>
+    `${tablePath(databaseName)}/${encodeURIComponent(tableName)}/rows`;
+  return {
+    listDatabases: generateListEndpoint(instance, path, map, false),
+    createDatabases: generateCreateEndpoint(instance, path, map),
+    deleteDatabases: generateDeleteEndpoint(instance, path, map),
+    listTables: (databaseName, query) => {
+      const endpoint = generateListEndpoint<ListRawTables, RawDBTable>(
+        instance,
+        tablePath(databaseName),
+        map,
+        false
+      );
+      return endpoint(query);
+    },
+    createTables: async (databaseName, items, ensureParent = false) => {
+      type Response = ItemsResponse<RawDBTable>;
+      const response = await rawRequest<Response>(instance, {
+        method: 'post',
+        url: tablePath(databaseName),
+        data: { items },
+        params: { ensureParent },
+      });
+      return map.addAndReturn(response.data.items, response);
+    },
+    deleteTables: (databaseName, items) => {
+      const endpoint = generateDeleteEndpoint<RawDBTable>(
+        instance,
+        tablePath(databaseName),
+        map
+      );
+      return endpoint(items);
+    },
+    listRows: (databaseName, tableName, query) => {
+      const endpoint = generateListEndpoint<ListRawRows, RawDBRow>(
+        instance,
+        rowPath(databaseName, tableName),
+        map,
+        false
+      );
+      if (query && query.onlyRowKeys) {
+        // @ts-ignore - we transform it
+        return endpoint({ columns: ',' });
```

Comment:
what does this comma do here?

Reply:
It says in the documentation: "Ordered list of column keys, separated by commas. Leave empty for all, use single comma to retrieve only row keys."